Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -172,27 +197,38 @@ STPPaymentCardTextFieldDelegate, MessageBannerViewControllerPresenting {
     self.viewModel.inputs.cardholderNameTextFieldReturn()
   }
 
- internal func paymentCardTextFieldDidChange(_ textField: STPPaymentCardTextField) {
+  // MARK: STPPaymentCardTextFieldDelegate
```

Comment:
Can we potentially update these pragma marks to separate content in the jump bar by using `//MARK: - Text`?